cmd/detectExecuteScan_test.go

```diff
@@ -20,15 +20,15 @@ type detectTestUtilsBundle struct {
 	*mock.FilesMock
 }
 
-func (c *detectTestUtilsBundle) RunExecutable(e string, p ...string) error {
+func (c *detectTestUtilsBundle) RunExecutable(string, ...string) error {
 	panic("not expected to be called in test")
 }
 
-func (c *detectTestUtilsBundle) SetOptions(options piperhttp.ClientOptions) {
+func (c *detectTestUtilsBundle) SetOptions(piperhttp.ClientOptions) {
```

Comment:
Can you skip the `_` parameter name here?

Reply:
yes, Goland reminded me that the parameter name is not used so it can be skipped.

Reply:
The `_` is only needed if at least one other parameter is used within the function.